### DISCUSSION TITLE
Add a shortcut to run code

### DISCUSSION THREAD

**Discussion by sbc640964** [Roboroads/laravel-tinker#367] [Feature requests]
Add a shortcut to run code

**Comment by Roboroads** [comment]
You mean the already available ctrl+shift+t?

**Comment by sbc640964** [comment]
@Roboroads, I mean that shortcut to run as this button
![image](https://github.com/user-attachments/assets/a722c512-2f8e-4476-ba73-63693be4669a)

**Reply by Roboroads** [reply]
Which is what ctrl+shift+t does 😊

That shortcut is the "universal smart action" - it guesses the thing you want to do:
- in a PHP file, it will open a tinker console
- if you select code in a php file, it will copy that code to a new tinker console and open it.
- if you press it in a tinker console, it will run the console, liek the button.

If you want to have individual shortcuts for all these actions, you can go into the ide settings on the keybindings menu and set them all up yourself 😊
